packages/gamut-patterns/src/patterns/stripe/StripeRegular.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { PatternComponent, PatternSvg } from '../../types';
+
+export const StripeRegular: React.FC<PatternComponent> = ({ ...props }) => (
+  <PatternSvg width="100%" height="100%" {...props} aria-hidden>
```

Comment:
You can make these default props on Pattern SVG so you don't have to repeat it every file :)
```suggestion
  <PatternSvg width="100%" height="100%" {...props} aria-hidden>
```